### PR TITLE
add test & solution of Unzipstring

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: solutions/unzipstring.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package solutions
+
+import "strconv"
+
+func Unzipstring(str string) string {
+    var result string
```
Comment (Hamzaelkhatri) [inline solutions/unzipstring.go:6]: add condition to check if the string is empty , also check if len of argument is less than 2 
``` go 
	if len(str) == 0 || len(str) < 2 {
		return "Invalid output"
	}
```

File: tests/unzipstring_test/main.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+
+func main() {
+	arr := []string{"6H8a", "3p6i1W", "2O5u2H0e", "2d", "2a 6p8f", "2p4;7g", "2t4dD", "82t4D", "2k5%9.", "", "5w6	2t",}
```
[the remark this answers is not in the document]
Comment (zainabdnaya) [reply tests/unzipstring_test/main.go:12]: number + whatever but not number

**Review by Hamzaelkhatri — APPROVED** [review]
LGTM!

File: tests/unzipstring_test/main.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
+	arr := []string{
```
Comment (MSilva95) [inline tests/unzipstring_test/main.go:11]: add some more test because the ones you have are the ones in the readme, its easy to cheat this way.

**Review by MSilva95 — CHANGES_REQUESTED** [review]

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**